### PR TITLE
Replace CNI's iptables.sh with istio-iptables golang implementation

### PR DESCRIPTION
**Pull request by jwendell**

### CODE REVIEW

**Comment by sdake** [comment]
> Why not directly call the istio-iptables functions from the CNI plugin's binary and install only that one binary?

@rlenglet - sounds like a good improvement - although perhaps a new piece of work.

This PR makes a simple improvement - so going to approve.

File: Makefile.core.mk [hunk 1]
```diff
@@ -280,7 +280,8 @@ BINARIES:=./istioctl/cmd/istioctl \
   ./pkg/test/echo/cmd/server \
   ./mixer/test/policybackend \
   ./operator/cmd/operator \
-  ./cni/cmd/istio-cni ./cni/cmd/istio-cni-repair
+  ./cni/cmd/istio-cni ./cni/cmd/istio-cni-repair \
+  ./tools/istio-iptables
```
Comment (howardjohn) [inline Makefile.core.mk:284]: hmm.. right now this is part of proxyv2, I guess the main() file is just left over. Its only 3 lines long though so maybe not a big concern

**Comment by jwendell** [comment]
> Why not directly call the istio-iptables functions from the CNI plugin's binary and install only that one binary?

Yeah, that would require more thought, as a binary is required to execute `nsenter`.

**Comment by rlenglet** [comment]
> Yeah, that would require more thought, as a binary is required to execute nsenter.

Good point.

**Pull request merged**